eks/modules/eks/worker-iam.tf

```diff
@@ -12,30 +12,40 @@ data "aws_iam_policy_document" "worker-role-policy" {
 resource "aws_iam_role" "aws-node" {
   name = "${aws_eks_cluster.aws.name}-worker-iam-role"
 
-  assume_role_policy = "${data.aws_iam_policy_document.worker-role-policy.json}"
+  assume_role_policy = data.aws_iam_policy_document.worker-role-policy.json
 }
 
 resource "aws_iam_role_policy_attachment" "aws-node-AmazonEKSWorkerNodePolicy" {
   policy_arn = "arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy"
-  role       = "${aws_iam_role.aws-node.name}"
+  role       = aws_iam_role.aws-node.name
 }
 
 resource "aws_iam_role_policy_attachment" "aws-node-AmazonEKS_CNI_Policy" {
   policy_arn = "arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy"
-  role       = "${aws_iam_role.aws-node.name}"
+  role       = aws_iam_role.aws-node.name
 }
 
 resource "aws_iam_role_policy_attachment" "aws-node-AmazonEC2ContainerRegistryReadOnly" {
   policy_arn = "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
-  role       = "${aws_iam_role.aws-node.name}"
+  role       = aws_iam_role.aws-node.name
 }
 
 #resource "aws_iam_role_policy_attachment" "worker-ext-dns-policy" {
```

Comment:
should we get rid of the commented stuff if its not needed?

Reply:
Removed.